tests/appsec/test_automated_login_events.py

```diff
@@ -367,11 +367,17 @@ def test_login_wrong_user_failure_local(self):
 
             assert meta["_dd.appsec.events.users.login.failure.auto.mode"] == "extended"
             assert meta["appsec.events.users.login.failure.track"] == "true"
-            if context.library != "dotnet":
-                assert meta["appsec.events.users.login.failure.usr.id"] == "invalidUser"
-            else:
+            if context.library == "ruby":
```

Comment:
Ideally checking these fields should be moved to a different test case, so that Ruby can signal this with `@missing_feature` for that case.